jobwatch/hourly_watch.py

```diff
@@ -21,6 +21,23 @@
 FILEDIR = os.path.dirname(__file__)
 
 
+def get_options():
+    parser = argparse.ArgumentParser(description='Replan Central monitor')
```

Comment:
The description is a leftover, apparently.

Reply:
Well, it is still *basically* a replan central monitor, but we've since added an "is kadi up" check and called it the hourly watch so...